components/CollectionCells/src/MDCCollectionViewCell.m

```diff
@@ -57,6 +57,8 @@ @implementation MDCCollectionViewCell {
   UIImageView *_editingSelectorImageView;
 }
 
+@synthesize inkView = _inkView;
```

Comment:
Ah, I was considering the generation of the methods, not the declaration of the variable.
LGTM.